### DISCUSSION TITLE
Build PLUS on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by Karl-Philippe** [PlusToolkit/PlusLib#1018] [Q&A]
Hi, 

I am trying to build PLUS in Ubuntu 22.04 LTS following those instructions:

> `sudo apt update && sudo apt install git build-essential cmake qt5-default qtmultimedia5-dev qttools5-dev libqt5xmlpatterns5-dev qtbase5-private-dev libqt5x11extras5-dev libxt-dev`
> `git clone git://github.com/PlusToolkit/PlusBuild.git`
> `mkdir PlusBuild-bin`
> `cd PlusBuild-bin`
> `cmake ../PlusBuild`
> `make -j8`

Howerver, when downloading the requirements I get the following error:
`E: Package 'qt5-default' has no installation candidate`

Have any of you already experienced this issue?
Do you have any suggestion to solve this?

Thank you,

**Comment by adamrankin** [comment]
From https://stackoverflow.com/a/74117077


Ubuntu 22.04 repository dont have qt5-default package.

You can get pre-built libraries by installing the qtbase5-dev.

```sudo apt install qtbase5-dev```
